pkg/visor/visor.go

```diff
@@ -358,68 +358,68 @@ func (node *Node) stopUnhandledApp(name string, pid int) {
 		return
 	}
 
-	node.logger.Infof("Found and killed hanged app %s with pid %d previously ran by this node", name, pid)
+	visor.logger.Infof("Found and killed hanged app %s with pid %d previously ran by this visor", name, pid)
 }
 
-// Close safely stops spawned Apps and messaging Node.
-func (node *Node) Close() (err error) {
-	if node == nil {
+// Close safely stops spawned Apps and messaging Visor.
```

Comment:
Not sure if it should be called `messaging Visor`

Reply:
This should not be called messaging visor. Probably just visor is best.